conf/common.cfg

```diff
@@ -0,0 +1,27 @@
+[db]
+# type of database
+backend_grid = django.db.backends.mysql
+backend_cloud = django.db.backends.mysql
+
+# host with database
+hostname_grid = localhost
+hostname_cloud = localhost
+
+# port to connect to
+port_grid = 3306
+port_cloud = 3306
+
+# database name
+name_grid =
+name_cloud =
+
+# database user
+username_grid = root
+username_cloud = root
+
+# password for database
+password_grid =
+password_cloud =
+
+[common]
+allowed_hosts =
```

Comment:
Should have a comment explaining that this is a comma separated list of hostnames.

Also, put this section at the top as it's the first option parsed.

Reply:
Moved this to top. Done